karma.conf.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const fs = require('fs')
+
+module.exports = function (karma) {
+  const glob = 'tests/browser/*.js'
+  const files = [{pattern: glob}]
+
+  const customLaunchers = {
+    SL_Edge: {
+      base: 'SauceLabs',
+      platform: 'Windows 10',
+      browserName: 'microsoftedge'
+    },
+    SL_Firefox: {
+      base: 'SauceLabs',
+      browserName: 'Firefox'
+    },
+    SL_Chrome: {
+      base: 'SauceLabs',
+      browserName: 'Chrome',
+      platform: 'Windows 10'
+    },
+    SL_Safari: {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OSX 10.11'
+    },
+    SL_IE11: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      version: '11'
+    },
+    SL_IE10: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      version: '10'
+    },
+    SL_IE9: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      version: '9'
+    }
+  }
+
+  const sauceConfig = {
+    sauceLabs: {
+      testName: 'choo browser tests',
+      recordScreenshots: false,
+      connectOptions: {
+        port: 5757,
+        logfile: 'sauce_connect.log'
+      },
+      public: 'public'
+    },
+    captureTimeout: 120000,
+    customLaunchers: customLaunchers,
+    browsers: Object.keys(customLaunchers)
+  }
+
+  let config = {
+    frameworks: ['tap', 'browserify'],
+    preprocessors: {
+      [glob]: ['browserify']
+    },
+    reporters: ['dots'],
+    port: 9876,
+    colors: true,
+    logLevel: karma.LOG_INFO,
+    files: files,
+    browserify: {
+      debug: true,
+      transform: ['sheetify/transform', 'es2020']
```

Comment:
D'oh!

Totally missed that one. :(. Thanks!

> On Jun 25, 2016, at 12:56, Yoshua Wuyts <notifications@github.com> wrote:
> 
> In karma.conf.js:
> 
> > +    browsers: Object.keys(customLaunchers)
> > +  }
> > +
> > +  let config = {
> > +    frameworks: ['tap', 'browserify'],
> > +    preprocessors: {
> > +      [glob]: ['browserify']
> > +    },
> > +    reporters: ['dots'],
> > +    port: 9876,
> > +    colors: true,
> > +    logLevel: karma.LOG_INFO,
> > +    files: files,
> > +    browserify: {
> > +      debug: true,
> > +      transform: ['sheetify/transform', 'es2020']
> ah yeah, es2020 should be under the "global" keyword here, as transform only applies to code declared in the repo ✨
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>